pkg/signature/kms/kms.go

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2021 The Sigstore Authors.
+// Copyright 2024 The Sigstore Authors.
```

Comment:
copyright should not be updated for an existing file.

Reply:
fixed